python/lsst/pipe/tasks/makeCoaddTempExp.py

```diff
@@ -100,7 +100,7 @@ class MakeCoaddTempExpConfig(CoaddBaseTask.ConfigClass):
     doApplyFinalizedPsf = pexConfig.Field(
         doc="Whether to apply finalized psf models and aperture correction map.",
         dtype=bool,
-        default=False,
+        default=True,
```

Comment:
Ok, so with this default, the user is expected to have run the `isolatedStarAssociation` and `finalizeCharacterization` tasks, right?  In the case where they haven't, what error message do they get?  I'm wondering if it's possible for `doApplyFinalizedPsf = True`, but in reality `finalizedPsfApCorrCatalog = None` and the correction silently just doesn't get applied (because `finalizedPsfApCorrCatalog` being not `None` is the condition that is checked in `prepareCalibratedExposures()`, without regard to the value of `doApplyFinalizedPsf`).

Reply:
Interesting question.  If running via `pipetask`, then the pipetask quantum graph generation will fail and say that required input connection `finalizedPsfApCorrCatalog` is not available.  If running the task directly then it will not get applied (but this is how all the external calibrations are used in `runQuantum`/`run`).

So I _think_ this is okay.